troubleshooting-scripts/kube-apiserver/check_apiserver-to-etcd.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+for i in $(docker exec -it kube-apiserver sh -c 'ps aux | grep kube-apiserver' | awk -F'--etcd-servers' '{print $2}' | awk -F ' ' '{print $1}' | tr -d '=' | tr , "\n" | sed '/^$/d' | awk -F '/' '{print $3}' | awk -F ':' '{print $1}')
```

Comment:
Possible alternative:

```suggestion
for i in $(docker inspect kube-apiserver | grep -m 1 "\--etcd-servers" | grep -Po '(?<=https://)[^:]*')
```